packages/ibm-products/src/components/Tearsheet/TearsheetShell.stories.jsx

```diff
@@ -179,6 +181,75 @@ NoAttributesSet.args = {
 };
 
 export const ReturnFocusToOpenButton = ReturnFocusTemplate.bind({});
-NoAttributesSet.args = {
+ReturnFocusToOpenButton.args = {
```

Comment:
I believe this story template to specify the `.args` was wrong, so updated this to `ReturnFocusToOpenButton`.